api/proto/Makefile

```diff
@@ -15,19 +15,14 @@
 # specific language governing permissions and limitations
 # under the License.
 
-
-VERSION ?= v1
-fbs := $(wildcard $(VERSION)/*.fbs)
-GO_FLAGS := --go-namespace $(VERSION) --gen-onefile
-JAVA_FLAGS := --gen-generated
+CC = protoc
 
 .PHONY: generate
 generate: generate-go
 
 .PHONY: generate-go
-generate-go: $(fbs)
-	flatc --go --grpc $(GO_FLAGS) -o $(VERSION) $^
-
-.PHONY: generate-java
-generate-java: $(fbs)
-	flatc --java --grpc $(JAVA_FLAGS) -o $(VERSION) $^
\ No newline at end of file
+generate-go:
```

Comment:
Would we leverage https://github.com/bufbuild/buf to check proto files and generate stubs?

Reply:
I've added `buf.yaml` and `buf.gen.yaml` to leverage `buf`.

I exclude two rules `RPC_REQUEST_STANDARD_NAME` and `RPC_RESPONSE_STANDARD_NAME` since they require `*Request` and `Response` for request and response types. Do we need to impose these two rules?